workflows/src/main/kotlin/com/r3/corda/lib/accounts/workflows/internal/schemas/AccountSchema.kt

```diff
@@ -7,8 +7,5 @@ object AccountSchema
 object AccountSchemaV1 : MappedSchema(
```

Comment:
You've got two objects that inherit from `MappedSchema` in the accounts codebase (I think) - this one and `AccountSchema`. Do you need both?